airbyte-integrations/connectors/source-lever-hiring/source_lever_hiring/manifest.yaml

```diff
@@ -0,0 +1,274 @@
+version: 0.50.2
+type: DeclarativeSource
+
+check:
+  type: CheckStream
+  stream_names:
+    - users
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path:
+        - data
+  paginator:
+    type: DefaultPaginator
+    page_token_option:
+      type: RequestOption
+      inject_into: request_parameter
+      field_name: offset
+    page_size_option:
+      type: RequestOption
+      field_name: limit
+      inject_into: request_parameter
+    pagination_strategy:
+      type: CursorPagination
+      page_size: 50
+      cursor_value: '{{ response.get("next", {}) }}'
+      stop_condition: '{{ not response.get("next", {}) }}'
+  incremental_sync:
+    type: DatetimeBasedCursor
+    cursor_field: updatedAt
+    cursor_datetime_formats:
+      - '%s'
+    datetime_format: '%s'
+    start_datetime:
+      type: MinMaxDatetime
+      datetime: '{{ config[''start_date''] }}'
+      datetime_format: '%Y-%m-%dT%H:%M:%SZ'
+    start_time_option:
+      type: RequestOption
+      field_name: updated_at_start
+      inject_into: request_parameter
+    end_datetime:
+      type: MinMaxDatetime
+      datetime: '{{ now_utc().strftime(''%Y-%m-%dT%H:%M:%SZ'') }}'
+      datetime_format: '%Y-%m-%dT%H:%M:%SZ'
+    step: PT100S
+    cursor_granularity: PT1S
+  opportunity_id_partition_router:
+    - type: SubstreamPartitionRouter
+      parent_stream_configs:
+        - type: ParentStreamConfig
+          parent_key: id
+          partition_field: opportunity_id
+          stream:
+            $ref: "#/definitions/opportunities_stream"
+  basic_authenticator:
+    type: BasicHttpAuthenticator
+    password: null
+    username: "{{ config['credentials']['api_key']  }}"
+  oauth2_authenticator:
+    type: OAuthAuthenticator
+    token_refresh_endpoint: https://{{ "sandbox-lever.auth0.com/" if config['environment'] = Sandbox else "auth.lever.co/" }}
+    client_id: "{{ config['credentials']['client_id'] }}"
+    client_secret: "{{ config['credentials']['client_secret'] }}"
+    refresh_token: "{{ config['credentials']['refresh_token'] }}"
+    grant_type: refresh_token
+  requester:
+    type: HttpRequester
+    url_base: https://api.{{ "sandbox." if config['environment'] = Sandbox }}lever.co/
+    http_method: GET
+    request_headers: {}
+    authenticator:
+      class_name: source_lever_hiring.components.AuthenticatorLeverHiring
+      basic_auth: "#/definitions/basic_authenticator"
+      oauth2: "#/definitions/oauth2_authenticator"
+    request_body_json: {}
+  opportunity_child_stream:
+    type: DeclarativeStream
+    name: '{{ parameters.name }}'
```

Comment:
It looks like this is causing some streams to be called {{ parameters.name }} when we run `discover`.

I'll solve this in my next commit by adding the extra key-value pairs the old-fashioned way under each child stream's `$ref`: https://docs.airbyte.com/connector-development/config-based/advanced-topics/#references